### PR TITLE
team 10 translation documents

### PR DESCRIPTION
**Pull request by caotaiming**
#24 group 10 translation

### CODE REVIEW

**Comment by zl117** [comment]
Comments from group 1:
  some minor mistakes for the long translation:
  1. Line 10: "an" internal toobox not "a"
  2. Line 40 : "it is even nessary" there is a typo, which should be necesary
  3. Line 40 : "container technology must have the compatibility of", we found no usage of this, normally compatible goes with "with", could be switched to "must be compatible with"
  4. Line 40 : "avoid develop" -> "avoid developing"
  5. Line 42 : "transition into" -> make a transition to , the original sentence lacks a verb
  6. Line 106 : "For most applications which are relied heavily on the operation system" should not use passive tense, should be "For most applications that rely heavily..."
  7. Line 142 : "support LXCFS implement" -> "support LXCFS to implement"
  If we are wrong, let us know, thank you!

**Comment by RuiteGuo** [comment]
This version is the earliest version. After review we have already fixed the majority issues you mentioned and we haven't been able to update it. By the way, transition itself is a verb.

**Comment by heranyu** [comment]
1. kubectl exec/attach/port-forward is a command
2. 第五点”.Then reply to the http request, “缺少主语，中间应该为，
3. After the Stream Server obtains the specified number of Streams, call the Container Manager's CreateExec and startExec methods in turn, performed the exec operation on the target container, and forwarded the IO stream to the corresponding stream. 这句话缺少主语

File: blog-en/6.  IO stream processing.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+origin doc link : https://github.com/pouchcontainer/blog/blob/master/blog-cn/PouchContainer%20CRI%E7%9A%84%E8%AE%BE%E8%AE%A1%E4%B8%8E%E5%AE%9E%E7%8E%B0.md
+
+
+### 6. IO stream processing
+
+Kubernetes provides functions such as kubectl exec/attach/port-forward to enable direct interaction between the user and a specific Pod or container, as showned below:
+
```
Comment (Letty5411) [inline blog-en/6.  IO stream processing.md]: typo : showned

File: blog-en/6.  IO stream processing.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    type ExecRequest struct {
+    	ContainerId string	// Execute the target container of exec
+    	Cmd []string	// Specific execution of the exec command
+    	Tty bool	// Whether to execute the exec command in a TTY
+    	Stdin bool	// Whether include Stdin flow
+    	Stdout bool	// Whether to include Stdout flow
+    	Stderr bool	// Whether to include Stderr flow 
+    }
+    ```
+2. Surprisingly, CRI Manager's `Exec` method does not directly call Container Manager and execute the exec command on the target container, but instead calls the built-in Stream Server's `GetExec`.
```
Comment (Letty5411) [inline blog-en/6.  IO stream processing.md]: remove `but`

File: blog-en/6.  IO stream processing.md (hunk 1 above)
Comment (caotaiming) [reply blog-en/6.  IO stream processing.md]: 原文中文就没有主语
很多技术性文档都有此类无主语句子只作为步骤性指导，用于描述动作，告诉读者怎样操作
不然每个句子前加一you can/ you should blah blah会让文档显得非常冗余

Comment (caotaiming) [reply blog-en/6.  IO stream processing.md]: 原文中文就没有主语
很多技术性文档都有此类无主语句子只作为步骤性指导，用于描述动作，告诉读者怎样操作
不然每个句子前加一you can/ you should blah blah会让文档显得非常冗余

**Comment by Starnop** [comment]
Too many commits to be able to merge.

**Pull request closed without merging**